packages/ui/src/primitives/connection.tsx

```diff
@@ -3,22 +3,31 @@ import * as React from 'react';
 
 import { Spinner } from './spinner';
 
-export const Root = React.forwardRef<HTMLDivElement, React.HTMLAttributes<HTMLDivElement>>(function Root(
-  { children, className, ...props },
-  forwardedRef,
-) {
+function getColumnCount(itemsLength: number, maxCols: number = 6): number {
```

Comment:
cc @austincalvelage (@alexcarpenter solved it!)

Reply:
Legend!